pkg/web/src/pages/Settings/cluster/EditClusterModal.tsx

```diff
@@ -85,14 +85,24 @@ export const EditClusterModal = React.memo(() => {
   const validateCraneUrl = (id: string) => {
     const res = { error: false, msg: '' };
     const cluster = clusters.find((cluster: any) => cluster.id === id);
-
+    let reg = /^(?=^.{3,255}$)(http(s)?:\/\/)?(www\.)?[a-zA-Z0-9][-a-zA-Z0-9]{0,62}(\.[a-zA-Z0-9][-a-zA-Z0-9]{0,62})+(:\d+)*(\/\w+\.\w+)*$/;
+    
+// console.log(`cluster`, cluster);
```

Comment:
Remove it

Reply:
Whether you use the regex for validation